.changes/unreleased/chart-operator-Removed-20250110-143018.yaml

```diff
@@ -0,0 +1,12 @@
+project: chart-operator
+kind: Deprecated
+body: |
+  The configurator values are not used in Operator Deployment anymore. If user would like to
+    set its own mirrored images (air-gapped environment), then specify them as `additionalCmdFlags`
+
+    ```yaml
+    additionalCmdFlags:
+    - --configurator-base-image=my.repo.com/configurator
```

Comment:
nit: include an example for setting the pull policy as well?

Reply:
ok